npm/webpack-dev-server/package.json

```diff
@@ -26,7 +26,7 @@
     "speed-measure-webpack-plugin": "1.4.2",
     "typescript": "^4.2.3",
     "webpack": "^4.44.2",
-    "webpack-dev-server": "^4.0.0"
+    "webpack-dev-server": "3.11.0"
```

Comment:
Doesn't matter too much but why did we need this change?

Reply:
I'm not sure, Might have been a result of me debugging some things, let me revert and see if anything breaks.

Reply:
Stuff broke 😆